scripts/ingests/ingest_utils.py

```diff
@@ -792,14 +805,21 @@ def ingest_photometry(db, sources, bands, magnitudes, magnitude_errors, referenc
 
```

Comment:
Anything that could possibly be `None`, I would do `Optional[Union[List[str], str]]` else `Union[List[str], str]]`